sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -2510,7 +2510,7 @@ export interface SearchRequest<TModel extends object = never> {
     skip?: number;
     speller?: QuerySpellerType;
     top?: number;
-    vector?: Vector<TModel>;
+    vectors?: Vector<TModel>[];
```

Comment:
should we mentioned this in Breaking Changes section of CHANGELOG?

Reply:
Yes, I'll add a line to tell users that the new option has a different name and array type.